startscript.js

```diff
@@ -0,0 +1 @@
+var cmd=require('node-cmd'); cmd.run('node server.js');
```

Comment:
I think this was something I was messing with for AWS deployment, and can be deleted unless you are using it somewhere.